libr/util/axml.c

```diff
@@ -306,7 +306,9 @@ R_API char *r_axml_decode(const ut8 *data, const ut64 data_size) {
 	RStrBuf *sb = NULL;
 	st32 depth = 0;
 
-	r_return_val_if_fail (data && data_size, NULL);
```

Comment:
Only check for data_size in here and keep the r_return for the data which never should be null